include/spdlog/details/os-inl.h

```diff
@@ -353,7 +360,22 @@ SPDLOG_INLINE size_t _thread_id() SPDLOG_NOEXCEPT
     return static_cast<size_t>(::thr_self());
 #elif __APPLE__
     uint64_t tid;
+    // There is no pthread_threadid_np prior to 10.6, and it is not supported on any PPC,
+    // including 10.6.8 Rosetta. __POWERPC__ is Apple-specific define encompassing ppc and ppc64.
+#    if (MAC_OS_X_VERSION_MAX_ALLOWED < 1060) || defined(__POWERPC__)
```

Comment:
Is this going to work with out iOS builds?

Reply:
Undefined identifiers are evaluated to 0.

Reply:
Updated to latest upstream master. No other notable changes since the previous update.